src/App.js

```diff
@@ -20,14 +21,42 @@ const StyledAppContainer = styled.div`
   height: 100vh;
 `
 
+const getTrailData = (trailFilename) => {
+  return fetch(`data/trails/${trailFilename}`).then((res) => res.json());
+};
```

Comment:
if you want to be even more concise:
```suggestion
const getTrailData = (trailFilename) => fetch(`data/trails/${trailFilename}`).then((res) => res.json());
```
using `()` instead of `{}` will return whatever is included within those brackets